src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx

```diff
@@ -47,4 +47,4 @@ const CalloutAspectTemplateStep: FC<CalloutAspectTemplateStepProps> = ({ callout
     />
   );
 };
-export default CalloutAspectTemplateStep;
+export default CalloutAspectTemplate;
```

Comment:
This component is not used anymore.

Reply:
It is going to be used when Callout templates are introduced. It will be used not as a step but as part of the Callout creation dialog. My logic was either delete it or prepare for future use.